java/code/src/com/redhat/rhn/taskomatic/task/HubReportDbUpdateWorker.java

```diff
@@ -76,8 +78,19 @@ public void setParentQueue(TaskQueue queue) {
         this.parentQueue = queue;
     }
 
+    private boolean checkRemoteData(Session remoteSession, String tableName) {
+        SelectMode query = generateExistsTable(remoteSession, tableName);
+        DataResult<Map<String, PGobject>> exist = query.execute();
+        return Boolean.parseBoolean(String.valueOf(exist.get(0).get("?column?")));
+    }
+
     private void updateRemoteData(Session remoteSession, Session localSession, String tableName, long mgmId) {
         TimeUtils.logTime(log, "Refreshing table " + tableName, () -> {
+            if (!checkRemoteData(remoteSession, tableName)) {
+                log.warn("Table " + tableName + " does not exist in server " +
+                        mgrServerInfo.getServer().getId());
```

Comment:
```suggestion
                log.warn("Table {} does not exist in server {}", tableName, mgrServerInfo.getServer().getId());
```
We should use formatting, instead of string concatenation now that we have migrated to log4j 2